omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -139,7 +139,8 @@
             chef_<%= node['private_chef']['opscode-erchef']['search_provider'] %>
         ]},
         {base_resource_url, <%= @helper.erl_atom_or_string(node['private_chef']['opscode-erchef']['base_resource_url']) %>},
-        {strict_search_result_acls, <%= @strict_search_result_acls %>}
+        {strict_search_result_acls, <%= @strict_search_result_acls %>},
+        {include_version_in_status, <%= @helper.to_booleam(node['private_chef']['opscode-erchef']['include_version_in_status'], true) %>}
```

Comment:
We can probably check for the values in chef_wm_status.erl. So this can be set to `node['private_chef']['opscode-erchef']['include_version_in_status']` without using the helper.

Reply:
ya that would be better